apis/flowmetrics/v1alpha1/flowmetric_webhook.go

```diff
@@ -0,0 +1,77 @@
+package v1alpha1
+
+import (
+	"context"
+	"fmt"
+	"github.com/netobserv/network-observability-operator/pkg/helper"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+// log is for logging in this package.
+var flowmetriclog = logf.Log.WithName("flowmetric-resource")
+
+type FlowMetricWebhook struct {
+	FlowMetric
+}
+
+// +kubebuilder:webhook:verbs=create;update,path=/validate-netobserv-io-v1alpha1-flowmetric,mutating=false,failurePolicy=fail,sideEffects=None,groups=netobserv.io,resources=flowmetrics,versions=v1alpha1,name=flowmetricvalidationwebhook.netobserv.io,admissionReviewVersions=v1
+// +kubebuilder:webhook:verbs=create;update,path=/validate-netobserv-io-v1beta2-flowcollector,mutating=false,failurePolicy=fail,groups=netobserv.io,resources=flowcollectors,versions=v1beta2,name=flowcollectorconversionwebhook.netobserv.io,sideEffects=None,admissionReviewVersions=v1
```

Comment:
was that line added automatically with operator-sdk?
The first line is for flowmetric, but the second is for flowcollector, for which we don't have a validation webhook, so should we remove it?

Reply:
w/o this 2nd line the generated manifest will have only one webhook so yes it seems related with conversion we can get both